openfisca_uk/parameters/dwp/pension_credit/guarantee_credit/carer/addition.yaml

```
@@ -0,0 +1,14 @@
+description: Addition to the Minimum Guarantee for each claimant who is a carer.
+values:
+  2020-04-01: 37.50
+  2021-04-01: 37.70
+  2022-04-01: 38.85
+metadata:
+  unit: currency-GBP
+  period: week
+  name: pc_carer_addition
+  label: PC carer addition
```

```suggestion
  label: Pension Credit carer addition
```
and elsewhere...I think the PolicyEngine app should avoid abbreviations, even at the cost of longer text